alerts/openshift-virtualization-operator/OrphanedVirtualMachineInstances.md

```diff
@@ -29,7 +29,7 @@ have a running `virt-handler` pod:
 3. Check the status of the `virt-handler` daemon:
 
    ```bash
-   $ oc get daemonset virt-handler --all-namespaces
+   $ oc get daemonsets --all-namespaces | grep virt-handler
```

Comment:
if you filter the output with grep, you need to update the example output.
but it will be better to use a selector, i.e. `-l kubevirt.io=virt-handler`